### ISSUE TITLE
Date format error

### ISSUE THREAD

**Issue by daaanny90** [icehaunter/vue3-datepicker#26]
I can't change the date format adding something else other than dd MM and yyyy.

Example to reproduce:
```
<Datepicker
      v-model="date"
      :locale="dateLang"
      inputFormat="eee dd.MM.yyyy"
  />
```

I get this errors in console:
```
[Vue warn]: Unhandled error during execution of watcher callback 
  at <Datepicker modelValue= Fri Feb 19 2021 09:05:04 GMT+0100 (Central European Standard Time) onUpdate:modelValue=fn locale= {code: "de", formatLong: {…}, localize: {…}, formatDistance: ƒ, formatRelative: ƒ, …}  ... > 
  at <DatePicker> 
  at <Speiseplan onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< undefined > > 
  at <RouterView> 
  at <App>

[Vue warn]: Unhandled error during execution of setup function 
  at <Datepicker modelValue= Fri Feb 19 2021 09:05:04 GMT+0100 (Central European Standard Time) onUpdate:modelValue=fn locale= {code: "de", formatLong: {…}, localize: {…}, formatDistance: ƒ, formatRelative: ƒ, …}  ... > 
  at <DatePicker> 
  at <Speiseplan onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< undefined > > 
  at <RouterView> 
  at <App>

[Vue warn]: Unhandled error during execution of scheduler flush. This is likely a Vue internals bug. Please open an issue at https://new-issue.vuejs.org/?repo=vuejs/vue-next 
  at <Datepicker modelValue= Fri Feb 19 2021 09:05:04 GMT+0100 (Central European Standard Time) onUpdate:modelValue=fn locale= {code: "de", formatLong: {…}, localize: {…}, formatDistance: ƒ, formatRelative: ƒ, …}  ... > 
  at <DatePicker> 
  at <Speiseplan onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< undefined > > 
  at <RouterView> 
  at <App>

Uncaught (in promise) RangeError: Format string contains an unescaped latin alphabet character `e`
    at eval (index.js?4bb6:109)
    at Array.map (<anonymous>)
    at lightFormat (index.js?4bb6:90)
    at eval (vue3-datepicker.esm.js?24e7:667)
    at callWithErrorHandling (runtime-core.esm-bundler.js?5c40:154)
    at getter (runtime-core.esm-bundler.js?5c40:2018)
    at reactiveEffect (reactivity.esm-bundler.js?a1e9:42)
    at doWatch (runtime-core.esm-bundler.js?5c40:2106)
    at watchEffect (runtime-core.esm-bundler.js?5c40:1950)
    at setup (vue3-datepicker.esm.js?24e7:665)
```

Am I doing something wrong or it is a bug?

**Comment by victorybiz** [comment]
Facing the same issue here, I can't change the `input-format` to other formats other than `dd` `MM` and `yyyy`.

**Comment by icehaunter** [comment]
Yeah, input format is using non-localized `lightFormat` ([docs](https://date-fns.org/v2.19.0/docs/lightFormat)) instead of the heavy `format` at the moment. My bad, I'll change it asap!

**Comment by icehaunter** [comment]
Fixed in #33
